taui/amplify/#current-cloud-backend/amplify-meta.json

```diff
@@ -0,0 +1,52 @@
+{
+	"providers": {
+		"awscloudformation": {
+			"AuthRoleName": "src-20190206161419-authRole",
+			"UnauthRoleArn": "arn:aws:iam::715496170458:role/src-20190206161419-unauthRole",
+			"AuthRoleArn": "arn:aws:iam::715496170458:role/src-20190206161419-authRole",
+			"Region": "us-east-1",
+			"DeploymentBucketName": "src-20190206161419-deployment",
+			"UnauthRoleName": "src-20190206161419-unauthRole",
+			"StackName": "src-20190206161419",
+			"StackId": "arn:aws:cloudformation:us-east-1:715496170458:stack/src-20190206161419/420d9450-2a2a-11e9-9b0b-12d3b8b76b62"
+		}
+	},
+	"auth": {
+		"cognitoEcholocatorTest": {
+			"service": "Cognito",
+			"providerPlugin": "awscloudformation",
+			"providerMetadata": {
+				"s3TemplateURL": "https://s3.amazonaws.com/src-20190206161419-deployment/amplify-cfn-templates/auth/cognitoEcholocatorTest-cloudformation-template.yml",
+				"logicalId": "authcognitoEcholocatorTest"
+			},
+			"lastPushTimeStamp": "2019-02-07T01:05:29.459Z",
+			"output": {
+				"AppClientSecret": "187m9erav4psm0bd76ut85sophhcnf6r7mhu3h3d904rcrdseltv",
```

Comment:
If this is the front-end (client) key, as I understand it, I think this should be fine; it will have to be in the static deployed site, and therefore exposed, anyways.

Reply:
Looking at it further, I'm not sure what this value is; there shouldn't be an app secret, just a client ID:
![image](https://user-images.githubusercontent.com/960264/52506780-a515e680-2bbd-11e9-867f-4f37579c98b3.png)

Reply:
Seems like the secret might be for mobile apps only: https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html Possibly a bug in Amplify, then, but if you're confident that it won't affect us I'm fine with leaving it.